test/e2e/run_volume_test.go

```diff
@@ -1122,4 +1122,13 @@ RUN chmod 755 /test1 /test2 /test3`, ALPINE)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
 	})
+
+	It("--mount flag defaults to volume if no type given", func() {
+		volName := "testvol"
+		podmanTest.PodmanExitCleanly("volume", "create", volName)
+
+		podmanTest.PodmanExitCleanly("run", "--rm", "--mount", fmt.Sprintf("src=%s,dest=/mnt", volName), ALPINE, "touch", "/mnt/testfile")
```

Comment:
I think this should be `target=/mnt` and the same one line below. I think this will fix the tests.

Reply:
I'm just going to add `dest` as a valid alias for `destination`. I cannot be the only one who messes this up.